### ISSUE TITLE
Failed to run in Azure Spring Apps

### ISSUE THREAD

**Issue by v-jiaodi** [Azure-Samples/spring-chatgpt-sample#5]
**Describe the issue:**
When executing the following command to deploy the .jar file for the app:
```
az spring app deploy \
   --resource-group ${RESOURCE_GROUP} \
   --service ${SERVICE_NAME} \
   --name ${APP_NAME} \
   --artifact-path spring-chatgpt-sample-webapi/target/spring-chatgpt-sample-webapi-0.0.1-SNAPSHOT.jar \
   --env AZURE_OPENAI_ENDPOINT=<your_azure_openai_endpoint> \    
         AZURE_OPENAI_APIKEY=<your_api_key> \
         AZURE_OPENAI_CHATDEPLOYMENTID=gpt-35-turbo \
         AZURE_OPENAI_EMBEDDINGDEPLOYMENTID=text-embedding-ada-002 \
         VECTORSTORE_FILE=/opt/spring-chatgpt-sample/doc_store.json \
   --runtime-version Java_17 \
   --jvm-options '-Xms1024m -Xmx2048m'
```
 the error is as follows:

![image](https://github.com/Azure-Samples/spring-chatgpt-sample/assets/80496810/6545ec4e-a4b7-4262-ae49-1fbce2c61ad5)

**Environment:**
- OS: Windows desktop.

**Expected behavior:**
  Deploying the .jar file for the app can pass.

 @hemarina for notification.

**Comment by yiliuTo** [comment]
@v-jiaodi are there any detailed logs about the internal server error? You can go to Azure portal for this deployment to get the deployment logs

**Comment by yiliuTo** [comment]
Hi @v-jiaodi any updates? besides, to get the deployment logs for your ASA deploy operation, you can go to your resource group in Azure portal and click the `Deployments` button:
![image](https://github.com/Azure-Samples/spring-chatgpt-sample/assets/63028776/aecf3594-03a8-4ad6-8dec-aa14b15b4360)

**Comment by v-jiaodi** [comment]
@yiliuTo We didn‘t see any error logs under `Deployments`.Do you want to see the following information？
![image](https://github.com/Azure-Samples/spring-chatgpt-sample/assets/80496810/9774a5f2-1121-472c-8be8-d559d0fca810)

**Comment by yiliuTo** [comment]
Close this issue since it has been resolved by #6
